src/midgard/util.cc

```diff
@@ -25,10 +25,6 @@
 
 namespace {
 
-constexpr double RAD_PER_METER = 1.0 / 6378160.187;
-constexpr double RAD_PER_DEG = valhalla::midgard::kPiDouble / 180.0;
-constexpr double DEG_PER_RAD = 180.0 / valhalla::midgard::kPiDouble;
```

Comment:
- `RAD_PER_DEG` and `DEG_PER_RAD` were unused. Moreover there're similar constants in `pointll.h`.
- moved `RAD_PER_METER` to `pointll.h` closer to the other constants